Src/DependencyCollector/Net45.Tests/packages.config

```diff
@@ -3,4 +3,5 @@
   <package id="Microsoft.ApplicationInsights" version="2.0.0-beta4" targetFramework="net451" />
```

Comment:
Not critical, but I wonder why other packages are targeted to `net451`. Any ideas?